geschaeftsordnung-fachschaftsrat.md

```diff
@@ -51,40 +51,39 @@ Diese Geschäftsordnung wurde am 29.03.2023 vom Fachschaftsrat beschlossen.
 
 ## § 5 Anwesenheit
 
-Bei der Anwesenheit von Mitgliedern des Fachschaftsrates während seiner Sitzungen wird unterschieden:
-
-1. Persönliche Anwesenheit: Alle Mitglieder des Fachschaftsrates, die sich während der Sitzung am Ort der Sitzung befinden, gelten als persönlich anwesend.
-2. Anwesenheit: Alle Mitglieder des Fachschaftsrates, die während der Sitzung persönlich anwesend sind oder per Telefon oder anderweitiger Einrichtung zur Kommunikation, insbesondere Internet- oder Videotelefonie, teilnehmen, gelten als anwesend.
+Alle Mitglieder des Fachschaftsrates, die sich während der Sitzung am Ort der Sitzung befinden oder per Telefon oder anderweitiger Einrichtung zur Kommunikation, insbesondere Internet- oder Videotelefonie, teilnehmen, gelten als anwesend.
 
 
 ## § 6 Sitzungen
 
 (1) Während der Vorlesungszeit soll gemäß Satzung der Fachschaft Digital Engineering jede Woche eine Sitzung stattfinden. In der vorlesungsfreien Zeit und in den akademischen Weihnachtsferien soll mindestens alle drei Wochen eine Sitzung stattfinden.
 
-(2) Der Vorsitz lädt zu den Sitzungen per E-Mail über einen öffentlichen Verteiler alle interessierten Mitglieder der Fachschaft ein.
+(2) Der Vorsitz lädt zu den Sitzungen per E-Mail über einen öffentlichen Verteiler alle interessierten Mitglieder der Fachschaft ein. Die Einladung soll am Vortag erfolgen.
+
+(3) Die Sitzung soll nach Möglichkeit in den Räumen der
```

Comment:
Puh, also das ist ein wildes juristisches Rabbit hole. Warum nicht einfach definieren, dass es ein der Fachschaft zugänglicher Raum am Campus ist.

Reply:
Und wie würdest du Campus sagen? Campus der DEF?

Reply:
Griebnitzsee?

Reply:
Campus Griebnitzsee ist in meiner Lesart UP ex DEF

Reply:
Hmm, sonst einfach Union bilden?

Reply:
Hmmm, wie genau sollen Gewerkschaften uns da jetzt helfen?

Reply:
Immer nur noch VV bei der GEW

Reply:
So angepasst